extensions/vscode/src/views/homeView.ts

```diff
@@ -680,12 +680,15 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
       return;
     }
 
-    const relPathPackageFile =
-      activeConfiguration.configuration.python?.packageFile;
-    if (relPathPackageFile === undefined) {
+    // Project is not configured for Python so cannot scan
+    // Scan button is not visible when this is the case
+    if (!activeConfiguration.configuration.python) {
```

Comment:
Double checking here, I'm not familiar with configuration defaults but is there the possibility for `activeConfiguration.configuration.python` to be an empty object at any point giving a false positive?

Reply:
I don't believe so. The schema is invalid if the Python configuration doesn't have a version, and when it only has that the `packageFile` and `packageManager` become the default value. So it cannot be an empty object without also being a configuration error.

The same goes for the R section